ui.frontend/src/components/CodeArgumentInput.tsx

```diff
@@ -188,14 +214,18 @@ const CodeArgumentInput: React.FC<CodeArgumentInputProps> = ({ arg }) => {
               );
             } else if (isMultiSelectArgument(arg)) {
               const display = arg.display === 'AUTO' ? (Object.entries(arg.options).length <= 3 ? 'CHECKBOX' : 'LIST') : arg.display;
+
               return display === 'CHECKBOX' ? (
                 <CheckboxGroup
-                  {...field}
```

Comment:
for me it was better because the side effects that I am not aware now have been resolved even being unaware ; now I feel like something new could not work